text_collector_examples/apcaccess.sh

```diff
@@ -107,6 +106,7 @@ while read line; do
 	var_val=$(echo $line | cut -d : -f 2- | awk '{$1=$1};1')
 
 	case "$var_key" in
+	DATE)		write_line 'node_textfile_mtime_seconds' $(convert_date "$var_val") 'gauge' 'Last time the daemon received data from the APC' ;;
```

Comment:
You don't need to add this, it's included by the node_exporter itself. :-)

Reply:
So is there a way to tell it how stale it is?

Reply:
Yes, the node_exporter automatically creates a `node_textfile_mtime_seconds` metric for every textfile.[0] This is automatic.

[0]: https://github.com/prometheus/node_exporter/blob/master/collector/textfile.go#L36-L41